react/features/browser-capabilities/capabilities.json

```diff
@@ -0,0 +1,71 @@
+{
+    "Safari": [
+        {"version": "6" },
+        {
+            "version": "10",
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": true
+            }
+        },
+        { }
+    ],
+    "Chrome":
+    [
+        {"version": "34" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": true
+            }
+        }
+    ],
+    "Opera":
+    [
+        {"version": "22" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": false
+            }
+        }
+    ],
+    "Firefox": [
+        {"version": "30" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": true
+            }
+        }
+    ],
+    "Internet Explorer": [
+        {"version": "10" },
+        {
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": true
+            }
+        }
+    ],
```

Comment:
What does this mean for IE10? Based on in the provided .md I would think this means IE 10 is not supported but IE11 has A/V support. Is that correct?

Reply:
Oh. Thanks for the note. It appears that I have been using the wrong compatibility matrix for Temasys. Acoording to https://confluence.temasys.com.sg/display/TWPP I should put 9+. Will fix it now!